assets/script.js

```diff
@@ -84,4 +84,78 @@ function getStreamingData(movieId) {
     });
 }
 
-// getStreamingData("tt0167260");
\ No newline at end of file
+// getStreamingData("tt0167260");
+
+
+// Gets movie data
+function getMovieData(movieId) {
+    var imdbApiUrl = "https://imdb-api.com/en/API/Title/k_zns86b2w/" + movieId;
+
+    fetch(imdbApiUrl).then(function (response) {
+        if (response.ok) {
+            response.json().then(function (data) {
+                console.log("--IMDb--");
+                console.log(data);
+                console.log(data.fullTitle);
+
+                let movieObject = {
+                    title: data.fullTitle,
+                    year: data.year,
+                    rating: data.imDbRating,
+                    image: data.image,
+                    plot: data.plot,
+                    streaming: getStreamingData(data.id)
+                };
+
+                console.log(movieObject);
+
+                return;
+            });
+        }
+    });
+}
+
+// add movie data to "movie-container"
+function addMovieData(movieObject) {
+    let movieContainer = document.querySelector("#movie-container");
+    let movieCard = document.createElement("div");
+    let movieImage = document.createElement("img");
+    let movieTitle = document.createElement("h2");
+    let movieYear = document.createElement("p");
+    let movieRating = document.createElement("p");
+    let moviePlot = document.createElement("p");
+    let movieStreaming = document.createElement("p");
+
+    movieCard.classList.add("movie-card");
+    movieImage.classList.add("movie-image");
+    movieTitle.classList.add("movie-title");
+    movieYear.classList.add("movie-year");
+    movieRating.classList.add("movie-rating");
+    moviePlot.classList.add("movie-plot");
+    movieStreaming.classList.add("movie-streaming");
+
+    movieImage.src = movieObject.image;
+    movieTitle.textContent = movieObject.title;
+    movieYear.textContent = movieObject.year;
+    movieRating.textContent = movieObject.rating;
+    moviePlot.textContent = movieObject.plot;
+    movieStreaming.textContent = movieObject.streaming;
+
+    movieCard.appendChild(movieImage);
+    movieCard.appendChild(movieTitle);
+    movieCard.appendChild(movieYear);
+    movieCard.appendChild(movieRating);
+    movieCard.appendChild(moviePlot);
+    movieCard.appendChild(movieStreaming);
```

Comment:
The order here is gunna be a pain to deal with. I think we need to go from the inside (of an element) out, then from top to bottom. Happy to help you with this part since it's my fault that it's so complicated.

I _think_ we'll wanna go in this order:
1. append the image to the figure element , then the ifgure to the movie card (figure appendChild movie image, movie card appendChild figure
2. append the rating icon to the rating ( rating appendChild icon)
3. append the the title to the top row div, then append the rating to the top row div
4. append the top row div to the content div
5. append the plot to the content div
6. append the content div to the movie description div
7. then a loop will run through the streaming array and create elements
8. append the streaming ul to the streaming div

Reply:
Sorry just realized, the last thing we should do is append the movie card to the movie container.